stylesheets/global.scss

```
@@ -48,17 +34,19 @@ a {
-    background-image: url("/static/images/external-link-white.svg");
-    background-position: 90% center;
-    padding-right: 0;
-  }
+.external::after {
+  content: "";
+  display: inline-block;
+  width: 0.7em;
+  height: 0.7em;
+  margin-left: 0.3em;
+  vertical-align: middle;
+  background-color: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)